src/main/java/com/gmail/woosay333/onlinebookstore/Application.java

```diff
@@ -0,0 +1,11 @@
+package com.gmail.woosay333.onlinebookstore;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application {
```

Comment:
It's ok and even a good practice to name the main class as the name of the application, as you had it before, or some `BookStoreApplication`